packages/portal-proto/src/features/set-operations/CountButtonWrapperForSetsAndCases.tsx

```diff
@@ -138,8 +138,8 @@ const CountButtonWrapperForSetsAndCases: React.FC<
   const createCohort = async () => {
     return await createSet({
       filters: filters,
-      intent: entityType == "cohort" ? "portal" : "user",
-      set_type: entityType == "cohort" ? "frozen" : "mutable",
+      intent: "portal",
```

Comment:
This function only ever gets called for the cohort entityType so this conditional isn't necessary.